app/src/js/components/oauth.js

```diff
@@ -1,53 +1,66 @@
-import React, { useState, useEffect } from 'react';
-import { connect } from 'react-redux';
-import withQueryParams from 'react-router-query-params';
-import { withRouter } from 'react-router-dom';
-import PropTypes from 'prop-types';
-import Modal from 'react-bootstrap/Modal';
-import get from 'lodash/get';
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import { Helmet } from 'react-helmet';
+import Modal from 'react-bootstrap/Modal';
+import PropTypes from 'prop-types';
+// import get from 'lodash/get';
```

Comment:
Reminder to remove commented code

Reply:
Darn! Thought I got them all. So many files. Lol

Reply:
Removed. Changes committed.